languoids/tree/atla1278/volt1241/nort3149/came1255/uban1244/zand1246/bara1360/ngal1296/md.ini

```diff
@@ -8,7 +8,11 @@ longitude = 24.61853
 macroareas = 
 	Africa
 countries = 
-	Sudan (SD)
+	Central African Republic (CF)
+	South Sudan (SS)
+links =
+	https://en.wikipedia.org/wiki/Ngala_language_(Zande)
+	https://www.wikidata.org/wiki/Q20707040
```

Comment:
With the wikipedia/wikidata links the curation story is like with WALS: We query wikidata periodically, see
https://github.com/glottolog/pyglottolog/blob/15e46bf283ba1100b46ae8f940f8e75cd85812d7/src/pyglottolog/links/wikidata.py#L9-L18
and find out what's linked to glottocodes on wikipedia. Since [Ngala on wikipedia](https://en.wikipedia.org/wiki/Ngala_language_(Zande)) has the appropriate glottocode, it should show up here automatically anyway. (So at least your manual addition here should survive an automatic update :) )

Reply:
Ok, good to know. Regarding Tagbo language: Is there any reason why this languoid is placed so far up in the north? This language is quite poorly documented and I haven't found any source for the exact location (other than DR Congo) where it is spoken (if it is still spoken at all).

Reply:
I found one: https://joshuaproject.net/languages/tbm which places it closer to Sere (which seems legit to me).